backend/src/modules/match-module/core/domain/Match.ts

```diff
@@ -15,6 +16,10 @@ export class Match {
     this.secondMatchSideId = props.secondMatchSideId;
     this.winner = props.winner;
   }
+
+  getLooser(winner: MatchSideId): MatchSideId | undefined {
```

Comment:
Lepiej unikać metod get / set :) Proponuję samo looser() (a nawet get looser(), skoro w TS mamy gettery). Cytat z Implementing Domain-Driven Design poniżej:

```
The simple but effective approach to object design keeps the Value Object faithful to the Ubiquitous Language. The use of getValuePercentage() is a technical computer statement, but valuePercentage() is a fluent human-readable language expression.
```


chociaż... ta metoda to już zupełny mindfuck :D Bo mam napisane get, a mam podać jakiś argument. 
Widziałbym bardziej, że np. w Match masz metode `wonBy(winner: MatchSideId): Match` a potem tylko robisz metode `get looser(): MatchSideId | undefined`.

Reply:
nareperuję :P